docs/authorization/scopes_and_consents/scopes.rst

```diff
@@ -195,6 +195,10 @@ Scope Reference
 
 .. autoclass:: ScopeCycleError
 
+.. autofunction:: scopes_to_str
+
+.. autofunction:: scopes_to_scope_list
```

Comment:
These utilities needs a bit more precursor explanation if they're  going in the primary scopes page. 

Maybe a dedicated subsection? Hard to say if that's overkill, but at least something disambiguating.

Reply:
I didn't want to worry about a full subsection, but I could do a rubric + a short paragraph.

Reply:
Yeah, even just a sentence informing that these are convenience utility functions would be good in my book.